### ISSUE TITLE
Doing fake upload with this service

### ISSUE THREAD

**Issue by MAMSEIN** [wangyu-/udp2raw-tunnel#508]
English Only.
Hello there
Thank you for your great project, it helps us who living in Iran to bypass GOV censorships.
We have a big issue with VPS in Iran.  we need to do fake upload to use unlimited data limitation.
I mean if we use 1 GB of download, we should upload at least 10 GB. 1/10   download/upload.
Can you add this option to your project?
You can see this project. it will add this option with command "--noise" https://github.com/radkesvat/ReverseTlsTunnel/ 
It has this option to do fake UDP upload over the connection between server and client.
But your service is very high quality and good for gaming so I need to have this option too to get unlimited data from Iranian VPS.
Thank you so much

**Comment by wangyu-** [comment]
no.  This feature is not relevant to udp2raw's focus. I won't implement this in udp2raw.

you can implement this in another tunnel, and chain with udp2raw:

wireguard client-->fake-upload tunnel client---->udp2raw client------------------------------>udp2raw server--->fake-upload tunnel server--->wireguard server

this way provides much better re-usability
